cohort-engine/src/main/java/com/ibm/cohort/engine/translation/InJVMCqlTranslationProvider.java

```diff
@@ -25,24 +26,37 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.ibm.cohort.engine.LibraryFormat;
+
 /**
- * Uses the CqlTranslator inprocess to convert CQL to ELM. This is
- * an alternative to using the CQL Translation Service microservice
- * which is a longer term goal and something we've tested with, but
- * are not ready to deliver.
+ * Uses the CqlTranslator inprocess to convert CQL to ELM. This is an
+ * alternative to using the CQL Translation Service microservice which is a
```

Comment:
Is the plan to eventually use a microservice still? I feel like I remember hearing that the existing CQL translation service wasn't going to meet our needs, but I couldn't remember if we were still writing our own at some point.

Reply:
Right now, there are no plans to use the microservice, but I'm keeping the indirection in case we do end up with some other translation mechanism. Could change if people want.